spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/catalog/TestTransformTableChange.java

```diff
@@ -58,6 +58,25 @@ void testTransformRenameColumn() {
     Assertions.assertEquals(newFiledName, gravitinoRenameColumn.getNewName());
   }
 
+  @Test
+  void testTransformUpdateColumnComment() {
+    String[] fieldNames = new String[] {"default_name"};
+    String newComment = "default_comment";
+
+    TableChange.UpdateColumnComment updateColumnComment =
+        (TableChange.UpdateColumnComment) TableChange.updateColumnComment(fieldNames, newComment);
+    com.datastrato.gravitino.rel.TableChange gravitinoChange =
+        GravitinoCatalog.transformTableChange(updateColumnComment);
+
+    Assertions.assertTrue(
+        gravitinoChange instanceof com.datastrato.gravitino.rel.TableChange.UpdateColumnComment);
+    com.datastrato.gravitino.rel.TableChange.UpdateColumnComment gravitinoUpdateColumnComment =
+        (com.datastrato.gravitino.rel.TableChange.UpdateColumnComment) gravitinoChange;
+
+    Assertions.assertEquals(fieldNames, gravitinoUpdateColumnComment.getFieldName());
```

Comment:
may be we should use Assertions.assertArrayEquals here

Reply:
Assertions.assertEquals using object.equals which for array it checks whether is the same object.
Assertions.assertArrayEquals checks whether the content of the array is equals.  this could pass test because the two array happens to be same array.  There're similar problem in other tests in `TestTransformTableChange`, could you fix them all?